tools/system_libs.py

```diff
@@ -489,18 +501,18 @@ class Dummy(object):
       for dep in value:
         shared.Settings.EXPORTED_FUNCTIONS.append('_' + dep)
 
-  system_libs = [('libcxx',        'a',  create_libcxx,      libcxx_symbols,      ['libcxxabi'], True),
-                 ('libcxxabi',     'bc', create_libcxxabi,   libcxxabi_symbols,   ['libc'],      False),
-                 ('gl',            'bc', create_gl,          gl_symbols,          ['libc'],      False),
-                 ('al',            'bc', create_al,          al_symbols,          ['libc'],      False),
-                 ('html5',         'bc', create_html5,       html5_symbols,       ['html5'],     False),
-                 ('compiler-rt',   'a',  create_compiler_rt, compiler_rt_symbols, ['libc'],      False),
-                 (malloc_name(),   'bc', create_malloc,      [],                  [],            False)]
+  system_libs = [('libcxx',        'a',  create_libcxx,      libcxx_symbols, ['libcxxabi'], True), # noqa
```

Comment:
what is "noqa"?

Reply:
It tells flake8 to ignore warnings for this line: http://flake8.pycqa.org/en/3.1.1/user/ignoring-errors.html